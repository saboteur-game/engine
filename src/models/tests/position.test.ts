import Position from "../position";

describe("Position", () => {
  it("can be serialized", () => {
    expect(new Position(1, 2).toJS()).toMatchSnapshot();
  });

  it("can be stringified", () => {
    expect(new Position(1, 2).toString()).toBe("1,2");
  });
});
