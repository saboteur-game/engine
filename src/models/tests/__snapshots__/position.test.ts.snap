// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Position can be serialized 1`] = `
Object {
  "id": "test-id-0",
  "x": 1,
  "y": 2,
}
`;
